docs/getting-started/install-zowe-sdks.md

```diff
@@ -0,0 +1,87 @@
+# Zowe SDKs installation
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Client Node.js SDK
+- Zowe Client Python SDK *technical preview*
+- Zowe Client Kotlin SDK *technical preview*
+- Zowe Client Java SDK *under development*
+
+## SDK documentation
+
+For detailed SDK documentation, see the following:
+- [Zowe Node.js SDK](https://docs.zowe.org/stable/typedoc/index.html)
+- [Zowe Client Python SDK](https://zowe-client-python-sdk.readthedocs.io/en/latest/)
```

Comment:
- [Zowe Client Kotlin SDK](https://for-mainframe.github.io/r2z/)
Comment: the link is providing the information for the other type of library, but is going to be reworked before the Kotlin SDK's GA. It is relevant since the Kotlin SDK is based on this library that is archived by now

Reply:
Hey @KUGDev,
If we plan to add the Kotlin SDK here, should we add the `[*techinal preview*]`?
![image](https://github.com/zowe/docs-site/assets/37381190/84003a98-3049-4dbf-920d-90d213d36fbe)

If so, I believe we should do the same for the python SDK. 😋 
![image](https://github.com/zowe/docs-site/assets/37381190/701bcd45-2515-46a9-803b-1bfe7c63d965)

Reply:
Hey @zFernand0 
For now it is relevant, I agree

Reply:
@KUGDev The link you provided gives me a 404. Is there another link we could use here?
![image](https://github.com/zowe/docs-site/assets/109533923/429e905d-e2c1-416b-9dc0-4bb52bbfcd64)

Reply:
@anaxceron for now you can use https://for-mainframe.github.io/r2z/